### DISCUSSION TITLE
Doc comment on Kubernetes (/installation/kubernetes)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1087] [Docs/blog comments]
# Doc comment on Kubernetes (/installation/kubernetes)

Make sure to set your desired Weaviate version.

https://weaviate.io/developers/weaviate/installation/kubernetes

**Comment by hughesadam87** [comment]
It would help me a lot if this guide showed how to actually hit/expose an endpoint after its deployed.  For example, I'm running it in docker-dekstop locally and I see the pods and services, but I don't know how to view the `/v1/Schema` endpoints in my browser.  With docker compose, it's simply on "localhost:8080" but have no clude how to expose it on k8s with default config?

**Reply by hughesadam87** [reply]
Looks like I was after port forwarding

    kubectl port-forward pods/weaviate-0 8080:8080

**Reply by valentinneher** [reply]
Thank you for the update!

**Comment by edwardbottom** [comment]
Is there support for an ingress with https/ssl support or plans to ship that with this helm chart or is that up to the user to configure?

**Reply by databyjp** [reply]
Hi, that's right - I believe those are outside the scope of this helm chart.
